.prow/provider-nutanix.yaml

```diff
@@ -15,6 +15,7 @@
 presubmits:
   - name: pre-kubermatic-e2e-nutanix-ubuntu-1.24
     decorate: true
+    optional: true
     run_if_changed: "(addons/csi/nutanix|pkg/provider/cloud/nutanix)"
```

Comment:
If there's no Nutanix setup, we probably shouldn't run those tests at all, so I'd drop this `run_if_changed`.

Reply:
Good point. Done.